lib/Twig/SimpleFunction.php

```diff
@@ -87,4 +89,14 @@ public function isVariadic()
     {
         return $this->options['is_variadic'];
     }
+
+    public function isDeprecated()
+    {
+        return $this->options['deprecated'];
+    }
+
+    public function getAlternative()
```

Comment:
these methods must also be added in ``Twig_Function_*`` (and the test and filter equivalent), otherwise you will get fatal errors when the deprecated classes are used to register them)